app/views/entities/_entity.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="<%= dom_id entity %>" class="entity">
+  <p>
+    <strong><%= entity.name %></strong>
+  </p>
+
+  <p>
+    <strong>$ <%= entity.amount %></strong>
+  </p>
+
```

Comment:
- [ ] There should be a date mentioned when the transaction was created as per design. Please add in the UI.

### Your design
![Screenshot from 2023-03-26 00-57-45](https://user-images.githubusercontent.com/26767383/227738133-d552a349-2e86-42a6-b4c3-495fd25982aa.png)


### Given design

![Screenshot from 2023-03-26 00-58-10](https://user-images.githubusercontent.com/26767383/227738126-b791f6be-73ff-4e47-9224-93046678599c.png)

Reply:
thanks to mention, it is fixed now